### PR TITLE
left hand home row!

### PR DESCRIPTION
**Pull request by kc0dhb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 helloworld
 ==========
 
+ASDF
```
Comment (kylebyerly-hp) [inline README.md:4]: asdf might be better

**Comment by kylebyerly-hp** [comment]
lkjh

**Comment by miqui** [comment]
LGTM

**Comment by kylebyerly-hp** [comment]
LGTRM

**Comment by kylebyerly-hp** [comment]
lgtm!

**Pull request merged**